docs/Installation.rst

```diff
@@ -114,147 +116,271 @@ This is only required if the unit tests are to be build.
+          PATH="/Library/Frameworks/Python.framework/Versions/3.13/bin:${PATH}"
+          export PATH
+
+          # Set shell environment for Homebrew.
+          eval "$(/usr/local/bin/brew shellenv)"
+
+          # Export pFUnit installation folder.
+          export PFUNIT_DIR=~/pFUnit/build/
+
+          # Export environmental variables required by pFUnit.
+          export F90=gfortran
+          export F90_VENDOR=GNU
+
+          # Export OpenMP root path and flags.
+          export OpenMP_ROOT=$(brew --prefix)/opt/libomp
+          export LDFLAGS="-L/usr/local/opt/libomp/lib"
+          export CPPFLAGS="-I/usr/local/opt/libomp/include"
+
+   * ARM::
+   
+          # Setting PATH for Python 3.13. The original version is saved in .zprofile.pysave.
+          PATH="/Library/Frameworks/Python.framework/Versions/3.13/bin:${PATH}"
+          export PATH
+
+          # Set shell environment for Homebrew.
+          eval "$(/opt/homebrew/bin/brew shellenv)"
+
+          # Export pFUnit installation folder.
+          export PFUNIT_DIR=~/pFUnit/build/
+
+          # Export environmental variables required by pFUnit.
+          export F90=gfortran
+          export F90_VENDOR=GNU
+
+          # Export OpenMP root path and flags.
+          export OpenMP_ROOT=$(brew --prefix)/opt/libomp
+          export LDFLAGS="-L/opt/homebrew/opt/libomp/lib"
+          export CPPFLAGS="-I/opt/homebrew/opt/libomp/include"
+
+#. Save the changes you made in your ``.zprofile`` file and close it. 
+   You may now hide hidden files (⌘ + ⇧ + .).
+
+#. Open a new `Terminal` window. By default, it should open in your `Home` directory, 
+   but if not navigate to it by entering::
+
+     cd
+
+#. Download the pFUnit repository from Git, enter the source code repository and 
+   create a build directory (e.g. build) by typing the following commands::
+
+	git clone https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
+	cd pFUnit
+	mkdir build
+	cd build
+
+#. Before proceeding, **make sure that the default C compiler is Apple Clang by entering 
+   the following command**::
+
+     gcc --version
+   
+   If it is not, then you have an alias (symlink) pointing to another C compiler. In this 
+   case, you have two options:
+   
+   * Remove the alias, which will default the C compiler back to Apple Clang for all future 
+     compilations. To do so, open a new `Finder` window then open the ‘Go to Folder’ prompt 
+     by pressing (⇧ + ⌘ + g) and entering /usr. Navigate to /local/bin, locate the `gcc` 
+     alias and delete it. Once this is done, you may revert to your `Terminal` window and type::
+   
+	gcc --version
+    
+     to ensure that the default C compiler is Apple Clang. Now initialise CMake (you should 
+     still be in the build folder on your `Terminal`) by typing::
+
+	cmake ./..
+
+   * Initialise CMake by specifying which C compiler to use. In your `Terminal` window enter 
+     the following::
+
+	cmake -D CMAKE_C_COMPILER=CLANG ./..
```

Comment:
Are you sure it  needs to be capitalised? I though it was supposed to be a path or command that invokes the compiler. Hence I was expecting `clang`...

Also I am not sure why are the steps with `gcc` above necessary. Sorry, I got a bit lost here.